src/Algebra/Properties/Monoid/Reasoning.agda

```diff
@@ -0,0 +1,68 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Equational reasoning for monoids
+-- (Utilities for identity and cancellation reasoning, extending semigroup reasoning)
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Bundles using (Monoid)
+
+module Algebra.Properties.Monoid.Reasoning {o ℓ} (M : Monoid o ℓ) where
+
+open Monoid M
+    using (Carrier; _∙_; _≈_; setoid; isMagma; semigroup; ε; sym; identityˡ
```

Comment:
Default indentation, here, as throughout the module is *2* spaces, not 4 as you have here.
Given that this is the `(n+1)`th time of making this observation, it suggests that you need to fix the defaults in your editor perhaps?